tests/test-cases/Group23-VIC-Machine-Service/23-03-VCH-Create.robot

```diff
@@ -54,10 +61,36 @@ Inspect VCH Config ${name}
 Get VCH ${name}
     Get Path Under Target    vch
     ${id}=    Run    echo '${OUTPUT}' | jq -r '.vchs[] | select(.name=="${name}").id'
+    Set Test Variable    ${id}
 
     Get Path Under Target    vch/${id}
 
 
+Pull Busy Box
+    [Arguments]    ${docker_params}    ${expected_rc}    ${expected_output}
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker -H ${docker_params} pull busybox
+
+    Should Be Equal As Integers    ${rc}    ${expected_rc}
+    Should Contain    ${output}    ${expected_output}
+
+
+Get Docker Host Params
+    [Arguments]    ${vch}    ${vch_name}
```

Comment:
As above; I think `vch` is the only valid value for `${vch}`, so it should be safe to remove the argument.

Reply:
Ok, will remove it.